src/Storefront/Resources/app/administration/src/modules/sw-theme-manager/page/sw-theme-manager-detail/index.js

```diff
@@ -542,9 +542,7 @@ Component.register('sw-theme-manager-detail', {
             this.removeInheritedFromChangeset(allValues);
 
             // Theme has to be reset, because inherited fields needs to be removed from the set
-            return this.themeService.resetTheme(this.themeId).then(() => {
-                return this.themeService.updateTheme(this.themeId, { config: allValues });
-            });
+            return this.themeService.updateTheme(this.themeId, { config: allValues }, { reset: 1 });
```

Comment:
```suggestion
            return this.themeService.updateTheme(this.themeId, { config: allValues }, { reset: true });
```

we should use real boolean values and then use `getBoolean` on the query in the PHP part

Reply:
did not know about that -- very good advice, thanks. I changed it